contracts/mechanics/garden-of-forking-paths/GardenOfForkingPaths.sol

```diff
@@ -327,23 +341,13 @@ contract GOFPFacet is
     function setStageRewards(
         uint256 sessionId,
         uint256[] calldata stages,
-        address[] calldata terminusAddresses,
-        uint256[] calldata terminusPoolIds,
-        uint256[] calldata rewardAmounts
+        Reward[] calldata rewards
     ) external onlyGameMaster {
         LibGOFP.GOFPStorage storage gs = LibGOFP.gofpStorage();
         require(
-            stages.length == terminusAddresses.length,
+            stages.length == rewards.length,
```

Comment:
Error message should mention `rewards`, not `terminusAddresses`.

Reply:
Fixed